### ISSUE TITLE
No Icons, Text instead.

### ISSUE THREAD

**Issue by Lvkz** [jsmarcus/Iconize#75]
Hi, I followed [#68](https://github.com/jsmarcus/Iconize/issues/68) to use the latest version of the plugin, but I couldn't make it work, for some reason, I'm only getting the text instead of the icon. 

![image](https://user-images.githubusercontent.com/2721391/42522473-0859e34a-8420-11e8-8a7e-3acc7862b5c0.png)

**Comment by natasha-robarge** [comment]
Hey, I'm also having issues with this- do you know when they may update the beta to incorporate Xamarin.Forms 3.0.1 or create a new stable release? Have you found a solution to this yet? If so, do you mind sharing what steps you took to fix it?

**Comment by Lvkz** [comment]
Nope, nothing yet, I used all the versions available of this project against all the Xamarin.Forms versions that I could match with, and nothing: just text instead of icons.

**Comment by natasha-robarge** [comment]
I ended up figuring it out - how did you install Iconize? Maybe I can help

**Comment by Lvkz** [comment]
Oh really?! Nice!! How did you make it work?

Among other things the last method I tried following the steps described in this thread: https://github.com/jsmarcus/Iconize/issues/68#issuecomment-388294021

**Comment by natasha-robarge** [comment]
In my MainActivity.cs file in the Android project I have: 

            Plugin.Iconize.Iconize.Init(ToolbarResource, TabLayoutResource);

Before the LoadApplication(new App());

I also have Plugin.Iconize.Iconize.With(new Plugin.Iconize.Fonts.FontAwesomeModule()); in my App.xaml,cs

And as far as Nuget packages go, I have the latest beta version/pre-release for all the packages I installed for Iconize

It worked after adding beta and the plugin line to my MainActivity.cs and my App.xaml.cs

**Comment by mo-kml** [comment]
hi guys
i still have this problem and none of these steps worked for me.
do you have any solution to this problem?
i`ve upgraded my nugets to latest version, i used this line but nothing worked

> Plugin.Iconize.Iconize.Init(ToolbarResource, TabLayoutResource);
any suggestions?

**Comment by jsmarcus** [comment]
You have to manually include the fonts. NuGet will no longer bring the font files into your app. Be sure to use the .ttf files and that they are named correctly. The easiest way would be to download them from the repository.

https://github.com/jsmarcus/Iconize/blob/master/src/Fonts/Plugin.Iconize.Material/iconize-material.ttf

**Comment by mo-kml** [comment]
Hey you
i`ve included the font to Asset folder in android and set the build action to AndroidAsset but still nothing.
i downloaded the font file from the link above.
do i need to do something else?

> You have to manually include the fonts. NuGet will no longer bring the font files into your app. Be sure to use the .ttf files and that they are named correctly. The easiest way would be to download them from the repository.
> 
> https://github.com/jsmarcus/Iconize/blob/master/src/Fonts/Plugin.Iconize.Material/iconize-material.ttf
